src/ngraph/pass/manager.cpp

```diff
@@ -55,6 +58,8 @@ void ngraph::pass::Manager::initialize_default_passes()
 
 void ngraph::pass::Manager::run_passes(shared_ptr<Function> func)
 {
+    bool profile_enabled = getenv("NGRAPH_PROFILE_PASS_ENABLE") != nullptr;
```

Comment:
we could probably make it `static const` to signify that this value won't change?

Reply:
const is ok, static is meh